test/browser/fixtures/index.html

```diff
@@ -0,0 +1,58 @@
3.74091],[-118.5199,34.02789],[-119.08099,34.07812],[-119.43878,34.34851],[-120.3678,34.44708],[-120.6228,34.60852],[-120.74432,35.15692],[-121.7146,36.16168],[-122.54749,37.55188],[-122.51202,37.78351],[-122.95319,38.11371],[-123.72711,38.95172],[-123.86511,39.76709],[-124.39801,40.31329],[-124.17877,41.14209],[-124.21368,41.99969],[-124.53278,42.76611],[-124.14209,43.7085],[-123.89893,45.5235],[-124.07959,46.86469],[-124.39569,47.72028],[-124.68719,48.18451],[-124.5661,48.3797],[-123.12,48.0401],[-122.58728,47.09613],[-122.34003,47.36011],[-122.5,48.18011],[-122.84003,49.00012],[-122.97418,49.00269],[-124.91022,49.98468],[-125.62457,50.41669],[-127.43561,50.83069],[-127.99268,51.71588],[-127.85028,52.32971],[-129.1297,52.75549],[-129.30518,53.56171],[-130.51489,54.28772],[-130.53607,54.80267],[-131.08582,55.17889],[-131.96722,55.49792],[-132.25,56.37012],[-133.53918,57.17889],[-134.078,58.12311],[-135.03821,58.18768],[-136.62799,58.21228],[-137.79999,58.50012],[-139.8678,59.5379],[-140.8252,59.72748],[-142.5744,60.08447],[-143.9588,59.99933],[-145.92548,60.45868],[-147.11438,60.8847],[-148.2243,60.6731],[-148.01801,59.97827],[-148.5708,59.91431],[-149.72778,59.70569],[-150.60822,59.36829],[-151.71631,59.15588],[-151.85938,59.74512],[-151.40973,60.72589],[-150.34692,61.03369],[-150.62109,61.28448],[-151.89581,60.72729],[-152.57831,60.06171],[-154.0191,59.35028],[-153.28748,58.86469],[-154.23248,58.14648],[-155.3075,57.72791],[-156.30829,57.42291],[-156.55609,56.9801],[-158.11719,56.46368],[-158.43329,55.99408],[-159.60327,55.56671],[-160.28967,55.64368],[-161.22302,55.36469],[-162.23779,55.02429],[-163.0694,54.68988],[-164.78558,54.4043],[-164.9422,54.57233],[-163.84833,55.03949],[-162.87,55.34808],[-161.8042,55.89508],[-160.5636,56.00812],[-160.0705,56.41809],[-158.68439,57.01672],[-158.46112,57.21692],[-157.72278,57.57013],[-157.55029,58.32831],[-157.04169,58.91888],[-158.1947,58.61591],[-158.51721,58.7879],[-159.05859,58.42432],[-159.71161,58.93152],[-159.9812,58.57269],[-160.35529,59.07111],[-161.35498,58.6709],[-161.96881,58.67169],[-162.05499,59.26691],[-161.87408,59.63373],[-162.51801,59.98969],[-163.8183,59.7981],[-164.66217,60.26752],[-165.34637,60.50751],[-165.35077,61.07391],[-166.1214,61.50012],[-165.73437,62.07507],[-164.91919,62.63312],[-164.5625,63.14648],[-163.7533,63.21948],[-163.0672,63.05951],[-162.2605,63.54187],[-161.53442,63.45587],[-160.77252,63.76611],[-160.95831,64.2229],[-161.51801,64.40289],[-160.77771,64.7887],[-161.39191,64.77728],[-162.453,64.55951],[-162.75781,64.33868],[-163.54639,64.55908],[-164.96082,64.44708],[-166.42529,64.68671],[-166.84497,65.08893],[-168.11047,65.6701],[-166.7052,66.08832],[-164.47467,66.57672],[-163.65253,66.57672],[-163.78851,66.07727],[-161.6778,66.11609],[-162.48969,66.73547],[-163.71973,67.11652],[-164.43091,67.61627],[-165.3902,68.04291],[-166.7644,68.35889],[-166.20471,68.88312],[-164.43079,68.91553],[-163.16858,69.37109],[-162.93048,69.85809],[-161.90887,70.33331],[-160.93481,70.447
```

Comment:
Can we move the map setup here (or at least the map options object) from the fixture into the test itself so that it's easier to add new tests with different setups and everything relevant is in one place?

Reply:
Moved the JSON to a separate file and the initialization to `browser.js` with the option of overriding individual keys.